src/cairo_run.rs

```
@@ -49,16 +49,21 @@ pub fn write_output(cairo_runner: &mut CairoRunner) -> Result<(), CairoRunError>
 
 /// Writes a trace as a binary file. Bincode encodes to little endian by default and each trace
 /// entry is composed of 3 usize values that are padded to always reach 64 bit size.
-pub fn write_binary_trace(relocated_trace: &[RelocatedTraceEntry], trace_file: &Path) {
-    let mut buffer = File::create(trace_file).expect("Error while creating trace file");
+pub fn write_binary_trace(
+    relocated_trace: &[RelocatedTraceEntry],
+    trace_file: &Path,
+) -> io::Result<()> {
+    let mut buffer = File::create(trace_file).expect("Could not create trace file");
```

This should return an error as well.